src/libs/actions/Report.ts

```diff
@@ -3665,6 +3666,16 @@ function prepareOnboardingOptimisticData(
             if (['addAccountingIntegration', 'setupCategoriesAndTags'].includes(task.type) && !userReportedIntegration) {
                 return false;
             }
+            type SkipViewTourOnboardingChoices = 'newDotSubmit' | 'newDotSplitChat' | 'newDotPersonalSpend' | 'newDotEmployer';
+            if (
+                task.type === 'viewTour' &&
+                [CONST.ONBOARDING_CHOICES.EMPLOYER, CONST.ONBOARDING_CHOICES.PERSONAL_SPEND, CONST.ONBOARDING_CHOICES.SUBMIT, CONST.ONBOARDING_CHOICES.CHAT_SPLIT].includes(
+                    introSelected?.choice as SkipViewTourOnboardingChoices,
+                ) &&
+                engagementChoice === CONST.ONBOARDING_CHOICES.MANAGE_TEAM
+            ) {
+                return false;
+            }
```

Comment:
Why do we need this block? As far as I can tell, none of the `CONST.ONBOARDING_MESSAGES` we use here have  a `viewTour` task?